debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnection.java

```diff
@@ -108,7 +108,7 @@ public void resetSessionToCdb() {
     }
 
     protected Set<TableId> getAllTableIds(String catalogName, String schemaNamePattern, boolean isView) throws SQLException {
-
+        schemaNamePattern = schemaNamePattern == null ? "" : schemaNamePattern;
```

Comment:
Wouldn't it make more sense to omit the `owner` part altogether if schema pattern is null? I guess the query optimizer will discard the %% clause anyways, but seems a bit cleaner in terms of our code. WDYT?

Reply:
Pushed one commit for that, and also for using a statement parameter (which we should have done so already before). I can't test though locally right now, so could you review and verify? Thx!

Reply:
The change LGTM, thanks!